scripts/features/cassandra.sh

```diff
@@ -20,8 +20,9 @@ touch /home/$WSL_USER_NAME/.homestead-features/cassandra
 chown -Rf $WSL_USER_NAME:$WSL_USER_GROUP /home/$WSL_USER_NAME/.homestead-features
 
 # Install Cassandra and driver dependencies
-echo "deb http://www.apache.org/dist/cassandra/debian 311x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
-wget -q -O - https://www.apache.org/dist/cassandra/KEYS | sudo apt-key add -
+echo "deb https://debian.cassandra.apache.org 41x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+curl https://downloads.apache.org/cassandra/KEYS | sudo apt-key add -
```

Comment:
i'm confused by this one? there's like 1000 keys

Reply:
also, I think we can drop the `-a`.

Reply:
This is one I was fighting with so I reverted to the docs' recommendation. There are a few of your comments that have the same scenario. Instead of reinventing the install just go w/ what the docs recommend.